src/Codeception/Lib/Parser.php

```diff
@@ -101,7 +101,7 @@ protected function addCommentStep($comment)
 
     public static function getClassesFromFile($file)
     {
-        include_once $file;
+        self::includeFile($file);
```

Comment:
I think anonymous function can be better here. At least it does just what needed - creates a new context without implementing a new method